lib/coinbase/faucet_transaction.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Coinbase
+  class FaucetTransaction
```

Comment:
These can be follow-ups but:
1. Yardocs
3. Can we add`wait!` and `status` functions as we do with Transfer. Possibly an opportunity to factor out an class or module that has those methods

Reply:
Added docs fro this class.

The `wait!` and `status` functions could be added, but this transaction is only returned after it lands on-chain.